### ISSUE TITLE
Adobe Comments on Structured Content

### ISSUE THREAD

**Issue by awkawk** [w3c/silver#405]
Concerned about “visually distinct” as a critical error –  this will require specific evaluation criteria. The WG needs to think about what is expected from the author vs what is able to be adapted with plugins or AT – for example, using a plugin it is possible to have headings be visually distinct in the exact way that a user prefers, will it be possible for authors to provide semantic content and rely on users use of such tools? 

Similarly, in WCAG 2.1 there are SC like text spacing and identify input purpose which expect that end users will have tools that allow them that adapt content, which we believe are beneficial and will help encourage the development of tools to support more users. We do not expect that WCAG would require page authors to build in support for all of the functionality that could be provided for these SC and believe that WCAG needs a clear position on this type of dependency for all types of semantic content, including headings.

**Comment by jpascalides** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by SuzanneTaylor** [comment]
Is this a fair summary for the portion of this comment that relates to the Requirements for WCAG 3?
"Adobe: Suggests that the structured content guideline shows a blending of requirements for content versus assistive technologies (AT) and that there should be a requirement to address how responsibilities are shared across guidelines such as semantic markup, text spacing, etc."

**Comment by awkawk** [comment]
I don't think that is exactly it. It might be that the structured content guideline shows a blending of requirements for content vs AT, but until we know more about how we will actually evaluate the criteria it is hard to say. Bottom line is that authors need to know precisely what is expected from them, and I'm not sure what that might mean in a "blend".

I also don't think that this is necessarily limited as a concept to structured content. If the Spacing SC from WCAG 2.1 is brought over, the same comment applies to what is an author responsibility and what is UA/AT.

**Comment by fstrr** [comment]
This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
